getting-started/r/index.md

```diff
@@ -216,7 +216,7 @@ jobs = list_jobs()
 jobs # printed as a tibble or data.frame, but the object is a list
 
 # or use the job id (in this example 'cZ2ND0Z5nhBFNQFq') as index to get a particular job overview
-jobs$cZ2ND0Z5nhBFNQFq
+list_jobs()$cZ2ND0Z5nhBFNQFq
```

Comment:
Why this? jobs is already defined, right?

Reply:
```suggestion
jobs$cZ2ND0Z5nhBFNQFq
```

Reply:
jobs is defined but a static object, so `jobs$ID` will return the same status without any updates.

Reply:
Yeah, but here it is only shown that this is an alternative way to get the details from the list_jobs() request, actually, earlier it says that you need to re-run list_jobs() to refresh the data. So this is good as it is IMHO.